ambry-messageformat/src/main/java/com.github.ambry.messageformat/BlobPropertiesSerDe.java

```diff
@@ -83,5 +86,7 @@ public static void serializeBlobProperties(ByteBuffer outputBuffer, BlobProperti
     Utils.serializeNullableString(outputBuffer, properties.getContentType());
```

Comment:
Not related to this PR, on line 80, do you know why we do not have sanity check for `outputBuffer`? e.g., it should be an empty buffer with a specific length.